### PR TITLE
docs: add information about proxy support

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
source of information: https://opentelemetry.io/docs/collector/configuration/#proxy-support

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -938,3 +939,29 @@ Flags:
       --set stringArray             Set arbitrary component config property. The component has to be defined in the config file and the flag has a higher precedence. Array config properties are overridden and maps are joined, note that only a single (first) array property can be set e.g. -set=processors.attributes.actions.key=some_key. Example --set=processors.batch.timeout=2s (default [])
   -v, --version                     version for otelcol-sumo
 ```
+
+## Proxy Support
+
+Exporters that leverage the HTTP communication (currently all) respect the following proxy environment variables:
```
Comment (sumo-drosiek) [inline docs/Configuration.md:945]: I think we can reword it to focus on our exporter only

Comment (kkujawa-sumo) [reply docs/Configuration.md]: Corrected, please check it ;)

**Review by pmalek-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]
